lib/src/track/local_video_track.dart

```diff
@@ -1,66 +1,66 @@
-import 'package:flutter_webrtc/flutter_webrtc.dart';
-
-import '../errors.dart';
-import 'options.dart';
-import 'video_track.dart';
+import '../imports.dart';
 
 /// A video track from the local device. Use static methods in this class to create
 /// video tracks.
 class LocalVideoTrack extends VideoTrack {
-  RTCRtpSender? get sender => transceiver?.sender;
+  //
+  // The latest options used for this track
+  //
+  LocalVideoTrackOptions? latestOptions;
```

Comment:
what's this used for?

Reply:
Since `mediaStreamTrack.getSettings` doesn't work, my goal was to get the actual resolution from the constraints used for `getUserMedia`

Reply:
Still figuring out how to make it more decent tho, I did make a PR for `mediaStreamTrack.getSettings` but it works only for web https://github.com/livekit/flutter-webrtc/pull/3 I'll merge it later.